spec/spec.md

```diff
@@ -0,0 +1,180 @@
+file.
+
+### Comment headers
+
+Each plain text file that can contain comments should contain comments at the
+top of the file (comment header) that declare that file's copyright and
+licensing information.
+
+If a file is not a plain text file or does not permit the inclusion of
+comments, the comment header that declares the file's copyright and licensing
+information should be in an adjacent file of the same name with the
+additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`).
+
+The comment header must contain one or more `SPDX-Copyright` tags, and one or
+more `SPDX-License-Information` tags. A tag is followed by a colon, followed by
+a text value, and terminated by a newline.
+
+The `SPDX-Copyright` tag must be followed by a copyright notice as described
+in <!-- FIXME: Refer to an internal section -->.
+
+Instead of the `SPDX-Copyright` tag, the symbols `Copyright`, `Copyright (c)`,
+`Copyright (C)`, `Copyright ©` or `©` may be used.
+
+The `SPDX-License-Identifier` tag must be followed by a valid SPDX License
+Expression describing the licensing of the file (example:
+`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`). If separate sections
+of the file are licensed differently, a different `SPDX-License-Identifier` tag
+should be included for each section.
+
+An example of a comment header:
+
+```
+# SPDX-Copyright: 2016, 2018-2019 Jane Doe <jane@example.com>
+# SPDX-Copyright: 2019 Example Company
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+```
+
+### DEP5
+
+Copyright and licensing information may be associated with a file through a DEP5
+file. The intended use case of this method is large directories where including
+a comment header in each file (or in `.license` companion files) is impossible
+or undesirable.
+
+The DEP5 file must be in `.reuse/dep5` from the root of the project.
+
+<!-- TODO: How to do multiple licenses without AND and OR. -->
+
+The `License` tag must be followed by a valid SPDX License Expression describing
+the licensing of the associated files.
+
+The `Copyright` tag must be followed by a copyright notice as described in <!--
+FIXME: Refer to an internal section -->.
+
+An example of a DEP5 file:
+
+```
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Project
+Upstream-Contact: Jane Doe <jane@example.com>
+Source: https://example.com/jane/project
+
+Files: po/*
+Copyright: 2019 Translation Company
+License: GPL-3.0-or-later
+```
+
+## Format of copyright notices
+
+A copyright notice must start with the copyright symbol (©), the word
+"Copyright", or an equivalent tag, followed by the year(s) of publication, the
+name of the copyright holder, and the e-mail address of the copyright holder.
+The order of these items may be changed, but should be consistent within the
+project. The items except the name of the copyright holder may be omitted.
+
+Examples of valid copyright notices:
```

Comment:
Again, ideally, I would like there to be only on valid copyright notice:

```
SPDX-Copyright: © 2019 Acorn Hacker <a.hacker@example.org>
SPDX-Copyright: © 2011 Some Company <https://www.somecompany.example>
```

Reply:
Big disagree.

This is for backwards compatibility. We really want REUSE to be compatible with what the Linux kernel is currently doing. The `SPDX-Copyright` tag is recommended with a SHOULD, and is exclusively mentioned in the tutorial (and I believe FAQ). The rest is just backwards compatibility.

Reply:
I have a very strong opinion on this, but won’t fight tooth and nail for it, as long as it also allows for the following to be REUSE valid (with or without an additional SPDX tag before it):

```
© 2019 Acorn Hacker <a.hacker@example.org>
© 2019 Muh Company <https://muh.example>
```l

Ultimately, the copyright field could be a free text in the Spec, and to have a (strong) sane recommendation in the Tutorial (and FAQ).

We can always make the 4.0 or 16.0 version of the Spec stricter in that regard, once the suggestions in the Tutorial and FAQ spread out more.